app/views/users/show.html.erb

```diff
@@ -1,7 +1,39 @@
 <h1> <%= @user.username.capitalize %>'s Profile </h1>
 <ul>
   <li>Country of origin: <%= @user.country_of_origin %></li>
-  <li>Gender: <%= @user.gender  %></li>
+  <% unless @user.gender == "Not specified" %>
```

Comment:
We would suggest that you define an instance method inside the `User` class that identifies whether the gender was unspecified or not. You don't want your views to have to know the internal implementation details of your model.

Consider:

```ruby
unless @user.unspecified_gender?
```

This will necessitate a model implementation and a correlating set of unit tests.